README.md

```diff
@@ -1,2 +1,20 @@
 # comp110-worksheet-C
 Base repository for COMP110 worksheet C
+
+a) the task is checking to see whether or not any elements are duplicated
+b) as they are both nested lists checking one element in list i compared to every element in list j if there was only one duplicate at the end of i with the end of j would be worst case scenario 
+
+c) because its starting at the same position and will terminate earlier 
+d) its approximately twice as fast as averaging out the checks to only checking once
```

Comment:
Right idea, but needs spelling out that the original algorithm compares each pair of elements twice hence the speed-up